src/components/Avatar/Examples.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import { StoryFn } from '@storybook/react';
+import { Sparkles as SparklesDrei } from '@react-three/drei';
+import type { Meta } from '@storybook/react';
+import React from 'react';
+import { Avatar as AvatarWrapper, CAMERA } from 'src/components/Avatar';
+import { getStoryAssetPath } from '../../services';
+import { AvatarProps } from './Avatar.component';
+
+const Avatar = (args: AvatarProps) => <AvatarWrapper {...args} />;
```

Comment:
Not sure why we need to wrap the original `Avatar` within another variable.

Reply:
it's just a wrapper dor the test component not exposed outside world